### PR TITLE
Fixing strict error

### PR DESCRIPTION
**Pull request by gelform**
In strict mode, the index `override-from` could not be found. It needs an `isset` to check for the index without error.

### CODE REVIEW

**Comment by pirogoeth** [comment]
@gelform Hey there, thanks for the PR! Does that usage of `$mailgun['override-from']` still throw errors with the newest 1.5.7 release? I added some code to the [config validation stuff](https://github.com/mailgun/wordpress-plugin/blob/master/includes/admin.php#L279) that should make sure that option gets added when you save. So if it does still throw errors with 1.5.7 could you try saving your config and then try sending again?

File: includes/wp-mail.php [hunk 1]
```diff
@@ -223,7 +223,7 @@ function wp_mail($to, $subject, $message, $headers = '', $attachments = array())
         }
     }
 
-    if ($mailgun['override-from'] && !empty($mailgun['from-name'])
+    if (isset($mailgun['override-from']) && !empty($mailgun['from-name'])
```
Comment (pirogoeth) [inline includes/wp-mail.php]: For this to work properly, the change should actually be `(isset($mailgun['override-from']) && $mailgun['override-from']) && ...`

**Comment by gelform** [comment]
Trying to save the config in 1.5.7 throws the notice `Notice: Undefined property: MailgunAdmin::$defaults in /Users/localhost/Sites/gelform/kanbanwp.com/kanbanwp.com/wp-content/plugins/mailgun/includes/admin.php on line 279` http://prntscr.com/dwj8u3

**Comment by pirogoeth** [comment]
@gelform Okay thank you I will push a fix ASAP.

**Comment by pirogoeth** [comment]
@gelform Can you try with the zipball from [this](https://github.com/mailgun/wordpress-plugin/tree/sjohn/defaults-fix) branch?

**Comment by gelform** [comment]
Saving the config using that branch worked!

**Comment by pirogoeth** [comment]
@gelform Yay! Thank you for testing. I'll push a bugfix release tomorrow :)

**Comment by pirogoeth** [comment]
@gelform I've included the revised proposed change from this PR in #42.

**Comment by gelform** [comment]
Thanks!

**Comment by pirogoeth** [comment]
@gelform No worries! Thanks for filing this PR/issue :)

**Pull request closed without merging**